v1/openapi/schemas/result.yaml

```diff
@@ -0,0 +1,47 @@
+x-component: 'schemas/result'
```

Comment:
This looks almost identical to schemas/request. Why don't we create a generic schema with all common fields, then inherit them into request and result structures, and expand them with an allOff array?

Reply:
I don't understand. Result and request have common fields. Do you propose to move common fields into a separate file? Does this make sense? Also, state and request have different fields. Do you offer one more separate file for them?

Reply:
@sansaian Yes. Move the common fields into a separate file, then create request and result schemas, inherit the common files with allOf, and expand request and result with non-common fields.

Reply:
Done